packages/api/schema/input.json

```diff
@@ -2,6 +2,66 @@
   "$ref": "#/definitions/ModelInput",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
+    "Actual": {
+      "additionalProperties": false,
+      "properties": {
+        "cases": {
+          "type": "number"
+        },
+        "cumulativeCases": {
+          "type": "number"
+        },
+        "cumulativeDeaths": {
+          "type": "number"
+        },
+        "date": {
+          "$ref": "#/definitions/ISODate"
+        },
+        "deaths": {
+          "type": "number"
+        },
+        "mobility": {
```

Comment:
Is mobility just a single number? Or do we need something more complex than this?